RP.Automation.API/Extensions.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Newtonsoft.Json;
+
+namespace RP.Automation.API
+{
+    public static class Extensions
```

Comment:
Class should e removed
You can use this single line of code in the appropriate place where u need it